python/ray/rllib/ppo/ppo.py

```diff
@@ -64,7 +64,7 @@
     "timesteps_per_batch": 4000,
     # Each tasks performs rollouts until at least this
     # number of steps is obtained
-    "min_steps_per_task": 1000,
+    "min_steps_per_task": 200,
```

Comment:
1000 seems to trip up too many experiments... 200 is a safer value and shouldn't be much of a hit